src/components/Header/TopNav/index.js

```diff
@@ -37,6 +39,8 @@ const TopNavList = styled.ul`
 
 const getSlugs = (arr = []) => arr.map(({ slug }) => slug).filter(i => i)
 
+const getColor = (arr, slug) => find(arr, { slug: slug }).logoColour
```

Comment:
`find` will return undefined if it can't find anything, can you do: 
`(find(arr, { slug }) || {}).logoColor`

Reply:
Fair point...it should always find something but let's just safe-guard. We never know 👍